server/plugins/permissions/util.go

```diff
@@ -0,0 +1,13 @@
+package permissions
+
+// sliceToMap is a helper function to convert a string slice to a map.
+func sliceToMap(s []string) map[string]bool {
```

Comment:
could you move and make it similar to
https://github.com/woodpecker-ci/woodpecker/blob/905bb9585371b3e86393d994f31983d936e24a91/shared/utils/slices.go#L53

?